File: problemas/1000/1036.dart
import 'dart:io';
import 'dart:math';

void main() {
  final entrada = stdin.readLineSync().split(' ');
  final a = double.parse(entrada[0]);
  final b = double.parse(entrada[1]);
  final c = double.parse(entrada[2]);
  final delta = b * b - 4 * a * c;

  if (delta < 0 || a == 0) {
    stdout.writeln('Impossivel calcular');
  } else {
    final r1 = (-b + sqrt(delta)) / (2 * a);
    final r2 = (-b - sqrt(delta)) / (2 * a);
    stdout.writeln('R1 = ${r1.toStringAsFixed(5)}');
    stdout.writeln('R2 = ${r2.toStringAsFixed(5)}');
  }
}

File: problemas/1000/1036.js
const [a, b, c] = require('fs')
  .readFileSync('/dev/stdin', 'utf-8')
  .split(' ')
  .map(x => Number(x));
const delta = b * b - 4.0 * a * c;

if (delta < 0 || a === 0) {
  console.log('Impossivel calcular');
} else {
  var r1 = (-b + Math.sqrt(delta)) / (2.0 * a);
  var r2 = (-b - Math.sqrt(delta)) / (2.0 * a);
  console.log(`R1 = ${r1.toFixed(5)}`);
  console.log(`R2 = ${r2.toFixed(5)}`);
}